examples/demo/README.md

```diff
@@ -0,0 +1,50 @@
+# React + TypeScript + Vite
```

Comment:
can we clean this readme to just say its for map component and not put all of this here?